src/content/docs/guides/configure-biome.mdx

```diff
@@ -0,0 +1,391 @@
+
+Biome is able to resolve configuration files from the `node_modules/` directory.
+So you can export your configuration file from a package, and import it in multiple projects.
+
+In order to do so, the first thing to do is to set up your "shared" Biome configuration in a certain way.
+Let's suppose that you want to share a configuration from a package called `@org/shared-configs`, using the specifier `@org/shared-configs/biome`.
+You have to create an `exports` entry in the `package.json` of this package:
+
+```json title="package.json" ins={5,3}
+{
+  "name": "@org/shared-configs",
+  "type": "module",
+  "exports": {
+    "./biome": "./biome.json"
+  }
+}
+```
+
+Make sure that `@org/shared-configs` is correctly installed in your project, and update the `biome.json` file to look like the following snippet:
+
+```json title="biome.json"
+{
+  "extends": ["@org/shared-configs/biome"]
+}
+```
+
+Biome will attempt to **resolve** your library `@org/shared-configs/` from your working directory.
+The working directory is:
+
+- when using the CLI, the directory where you execute your scripts from.
+  Usually it matches the location of your `package.json` file;
+- when using the LSP, the root directory of your project.
+
+:::caution
+To avoid a breaking change with how the existing resolution works, paths starting with a dot `.` or ending with `.json` or `.jsonc` **won't** be resolved from `node_modules/`.
+:::
+
+For more information about the resolution algorithm, refer to the [Node.js documentation](https://nodejs.org/api/esm.html#resolution-and-loading-algorithm).
+
+
+## Ignoring files
+
+The first way to control which files and directories are processed by Biome is to list them in the CLI.
+In the following command, we format only `file1.js` and all the files in the `src` directory.
+The directories are recursively traversed.
+
+```shell
+biome format file1.js src/
+```
+
+:::caution
+Glob patterns used on the command line are not interpreted by Biome.
+They are expanded by your shell.
+Some shells don't support the recursive glob `**`.
+:::
+
+The Biome configuration file can be used to refine which files are processed.
+You can explicitly list the files to be processed using `include` and the files not to be processed using `ignore`.
+`include` and `ignore` accepts globs patterns such as `src/**/*.js`.
+See the related section for which glob syntaxes are supported.
+`include` is always applied first before applying `ignore`.
+This allows you to include some files and to ignore some of the file you included.
+
+:::note
+`include` and `ignore` have a slightly different semantics.
+`ignore` applies to both regular files and folders,
+while `include` is only tested on regular files.
+This means that if you want to prevent Biome from traversing a folder,
+you have to add the folder to `ignore`.
+This is a current limitation that we will fix in a future release.
```

Comment:
To remove. No need to explain future stuff, because we don't know when it will change and how. And it forces us to come back here and remove it.